snooty.toml

```diff
@@ -2,29 +2,75 @@ name = "c"
 title = "C Driver"
 
 intersphinx = [ "https://www.mongodb.com/docs/manual/objects.inv",
-                "https://www.mongodb.com/docs/atlas/objects.inv"
+                "https://www.mongodb.com/docs/atlas/objects.inv",
+                "https://www.mongodb.com/docs/drivers/objects.inv",
               ]
 
 toc_landing_pages = [
-    "/source/docs-libmongoc/api",
-    "/source/docs-libmongoc/api/errors",
-    "/source/docs-libmongoc/api/init-cleanup",
-    "/source/docs-libmongoc/api/mongoc_auto_encryption_opts_t",
-    "/source/docs-libmongoc/api/mongoc_bulk_operation_t",
-    "/source/docs-libmongoc/api/mongoc_change_stream_t",
-    "/source/docs-libmongoc/api/mongoc_client_encryption_t",
-    "/source/docs-libmongoc/application-performance-monitoring",
-    "/source/docs-libmongoc/guides",
-    "/source/docs-libmongoc/howto",
-    "/source/docs-libmongoc/includes",
-    "/source/docs-libmongoc/ref",
-    "/source/docs-libmongoc/static",
-    "/source/docs-libmongoc/tutorials",
+    "/libbson",
+    "/libbson/tutorials",
+    "/libbson/guides",
+    "/libbson/cross-platform-notes",
+    "/libbson/api",
```

Comment:
Confirming: should the API documentation be migrated?

I expect the C team will own the API documentation. Maybe the "API Reference" sections can link to:

- https://mongoc.org/libbson/current/api.html
- https://mongoc.org/libmongoc/current/api.html

Reply:
API docs removed. Resolving.